server/src/test/scala/services/PortStateSpec.scala

```diff
@@ -288,4 +293,32 @@ class PortStateSpec extends CrunchTestLike {
   private def arrivalsToFlightsWithSplits(arrivals: List[Arrival]): List[ApiFlightWithSplits] = {
     arrivals.map(a => ApiFlightWithSplits(a, Set()))
   }
+
+  "Given a 'slow' live crunch actor containing a single flight which takes 2 seconds to provide its state to the PortStateActor" >> {
+    "When I immediately ask the PortStateActor for flights for the day the crunch actor's flight is scheduled" >> {
+      "Then the response should contain the crunch actor's flight" >> {
+        val today = SDate("2020-07-03T00:00")
+        val flightsWithSplits = ApiFlightWithSplits(ArrivalGenerator.arrival(iata = "BA0001", schDt = today.toISOString()), Set())
+        val maybePortState = Option(PortState(Iterable(flightsWithSplits), Iterable(), Iterable()))
+        val liveActor = Props(new SlowCrunchStateActor(maybePortState, delay = 2 seconds))
+        val fcstActor = Props(new SlowCrunchStateActor(Option(PortState.empty), delay = 0 seconds))
+        val portStateActor = system.actorOf(Props(new PortStateActor(liveActor, fcstActor, () => today, 2, defaultAirportConfig.queuesByTerminal)))
```

Comment:
is it worth making this delay shorter to prevent slowing the tests down by 2 seconds?

Reply:
Sure. I've reduced it to 1 second